### ISSUE TITLE
I2I: Create an AMP component which embed our Connatix Story Player

### ISSUE THREAD

**Issue by rares-mihai-preda** [ampproject/amphtml#26593]
## Summary

Create an AMP component that embeds Connatix's story player. We wanted to create an AMP component for our publishers in order to easily integrate this into their amp pages.
We chose to implement a custom element instead of using an amp-iframe in order to have clear product visibility for our customers and differentiate between our Elements Player which already is a custom amp element. 

This will be used by many of our direct clients which use Connatix Playspace Player also in their non-amp pages. This is a publisher oriented product that Connatix brings to the market and it will be used in many different sites and domains by any one of our clients.  

## Design document
https://connatix.com/playspace

## Motivation

The pull request  (https://github.com/ampproject/amphtml/pull/26592 ) doesn't affect any other AMP file. It is a new component that embeds our code in an iframe and it was added to the bundles.config.js

### Component's custom attributes
**data-player-id (required)** ->Connatix story player id. This can be found at the Story Players section in the Connatix management dashboard (ex: https://playspace.connatix.com/story-player)
**data-orientation (required)** -> Orientation Mode: "portrait" or "landscape"
**data-story-id (optional)** -> Connatix story id. This can be found at the Story section in the Connatix management dashboard (ex: https://playspace.connatix.com/story)

### Component's layouts
Only the responsive layout is supported by this component because our story player has two ratios: 4/5 on portrait and 16/9 plus another 72px on landscape orientation.

## Additional context
More details: https://connatix.com/playspace

Link to the document to be embedded:
view-source:https://cds.connatix.com/p/plugins/connatix.playspace.embed.html

We use postMessage to send to our embedded document that the viewport size changed and pause event.

The embedded send the information to our story player. The viewport size is sent in our story player in order to know if the story player should change the orientation. Our story player has a mixed layout that requires a minimum viewport width and it can be configured from our back office. Under that viewport width the player is rendered in portrait otherwise is rendered in the landscape. 
The pause event is used to pause our story player.
/cc @alanorozco

**Comment by wassgha** [comment]
@alanorozco even though this is not a player per se, do you think it can be solved by the declarative video players proposal?

**Comment by alanorozco** [comment]
@wassgha Insofar as this possibly being a configuration of an aliased `amp-iframe`-like element, yes.

@rares-mihai-preda 

Could you specify projected use for this component? How many partners do you expect will use it?

Additionally, please:

- Specify goal of the component more clearly: **Why can't this be an amp-iframe**?
- Include an example URL to a document to be embedded.
- Specify `postMessage` interface between embedded document and host.
- Specify exposed component attributes and clarify.
- Specify component layouts and clarify.

**Comment by rares-mihai-preda** [comment]
> @wassgha
> 
> Insofar as this possibly being a configuration of an aliased `amp-iframe`-like element, yes. But let's put that aside while we discuss requirements :)
> 
> @rares-mihai-preda
> 
> Could you specify projected use for this component? How many partners or authors do you expect will use it? How many domains?
> 
> Additionally, please:
> 
> * Specify goal of the component more clearly: **Why can't this be an amp-iframe**?
> * Include an example URL to a document to be embedded.
> * Specify `postMessage` interface between embedded document and host.
> * Specify exposed component attributes and clarify.
> * Specify component layouts and clarify.

@alanorozco We updated this l2l description. Please review it again and let us know if there is any other missing information.

**Comment by rares-mihai-preda** [comment]
l2l closed. Details: Design Review 2020-04-08 20:00 UTC (Connatix Story Player, AMP testing & release quality, wg-performance) #27599
